terraform/hetzner/servers.tf

```diff
@@ -29,6 +29,8 @@ resource "hcloud_server" "seed" {
   placement_group_id = hcloud_placement_group.seed.id
   user_data          = templatefile("./templates/seed_user_data.tpl", var.seed_user_data)
   backups            = var.network == "testnet" || var.network == "mainnet" ? true : false
+  delete_protection  = var.network == "testnet" || var.network == "mainnet" ? true : false
```

Comment:
@nikola-cheqd would it make sense to just have a variable that sets this? and then just use that everywhere:
```hcl
variable enable_delete_protection {
    default = true
    type = boolean
}
```